modules/core/src/lib/layer.ts

```diff
@@ -1107,6 +1109,12 @@ export default abstract class Layer<PropsT extends {} = {}> extends Component<
           lightSources
         } = moduleParameters;
 
+        const maskProps = {
```

Comment:
Type the props?

Reply:
The type is in `deck.gl/extensions` and it is an open question whether this is the correct place to update the `ShaderInputs` as in many cases the referenced `ShaderModules` will not be present. Agree that we should type, will revisit when cleaning up, e.g. moving the `shaderInputs.setProps` into the extensions